imageroot/systemd/user/nginx.service

```
@@ -14,7 +14,7 @@ ExecStart=/usr/bin/podman run --conmon-pidfile %t/nginx.pid \
     --cidfile %t/nginx.ctr-id --cgroups=no-conmon \
     --pod-id-file %t/webserver.pod-id --replace -d --name  nginx \
     --volume ./conf.d:/etc/nginx/conf.d:z --volume websites:/usr/share/nginx/html:Z \
-    docker.io/nginx:1.20.2
+    docker.io/${NGINX_IMAGE}
```

It should be without prefix; if not, there's something odd...

```suggestion
    ${NGINX_IMAGE}
```

---

done, it was friday, thank again